packages/module/package.json

```diff
@@ -31,7 +31,9 @@
     "branches": "main"
   },
   "publishConfig": {
-    "access": "public"
+    "access": "public",
+    "registry": "https://registry.npmjs.org/",
```

Comment:
Something I'm just curious about, not sure if this `registry` key is needed? I'm not using it in catalog view, but maybe I should be.